build.sh

```diff
@@ -2,8 +2,8 @@
 
 usage() { echo "Usage: $0 [-t <tag>] [-g <geomesa.version>] [-a <accumulo.version>]" 1>&2; exit 1; }
 
-ACCUMULO_VERSION=1.7.2
-GEOMESA_VERSION=1.3.0-m3-SNAPSHOT
+ACCUMULO_VERSION=1.8.0
```

Comment:
My preference would be to stick to Accumulo 1.7.2.  GeoMesa 1.3.x is built against it.  User needing to use Accumulo 1.8.0 need to build GeoMesa with a profile.